restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -47,19 +51,22 @@
   private final ObjectMapper mapper;
   private final DocsSchemaChecker schemaChecker;
   private final JsonSchemaHandler jsonSchemaHandler;
+  private final ReactiveDocumentService reactiveDocumentService;
```

Comment:
ohh no, let's not bind these two as dependencies.. as said it would be better to implement everything in the `ReactiveDocumentService` and `ReactiveDocumentResourceV2`